stylesheets/commons/PanelBox/Heroes.less

```diff
@@ -0,0 +1,163 @@
+.heroes-panel {
+	&__category {
+		&-title {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			border-bottom: 0.5rem solid;
+			padding: 0.5rem 0;
+			margin-bottom: 0.25rem;
+
+			.category--strength & {
+				.theme--light & {
+					border-bottom-color: var( --clr-california-40 );
+				}
+
+				.theme--dark & {
+					border-bottom-color: var( --clr-california-80 );
+				}
+			}
+
+			.category--agility & {
+				.theme--light & {
+					border-bottom-color: var( --clr-semantic-positive-40 );
+				}
+
+				.theme--dark & {
+					border-bottom-color: var( --clr-semantic-positive-80 );
+				}
+			}
+
+			.category--intelligence & {
+				.theme--light & {
+					border-bottom-color: var( --clr-sapphire-40 );
+				}
+
+				.theme--dark & {
+					border-bottom-color: var( --clr-sapphire-80 );
+				}
+			}
+
+			.category--universal & {
+				.theme--light & {
+					border-bottom-color: var( --clr-gigas-40 );
+				}
+
+				.theme--dark & {
+					border-bottom-color: var( --clr-gigas-80 );
+				}
+			}
+
+			.category--upcoming & {
+				.theme--light & {
+					border-bottom-color: var( --clr-secondary-25 );
+				}
+
+				.theme--dark & {
+					border-bottom-color: var( --clr-secondary-100 );
+				}
+			}
+
+			> a {
+				font-weight: bold;
+				margin-right: 0.5rem;
+			}
+		}
+
+		ul&-list {
+			@_gutter-y: 0.5rem;
+			@_gutter-x: 0.5rem;
+			display: flex;
+			flex-wrap: wrap;
+			justify-content: center;
+			list-style: none;
+			margin: calc( -0.5 * @_gutter-y ) calc( -0.5 * @_gutter-x ) 0.5rem calc( -0.5 * @_gutter-x );
+
+			> li {
+				flex: 0 0 auto;
+				width: calc( 100% / 3 );
+				max-width: 100%;
+				padding-right: calc( @_gutter-x * 0.5 );
+				padding-left: calc( @_gutter-x * 0.5 );
+				margin-top: @_gutter-y;
```

Comment:
Would be better using cell/row padding or spacing no? Feels like adding custom padding is slightly overkilled

Reply:
I tried to do it with `gap` at first but I couldn't get the children to perfectly align to the left and the right of the parent container for some reason.
For example if I do `gap: 8px`,  the child css would be `width: calc( 12.5% - 7px)` to make it fit 8 items on a row on desktop. If tou would change that to `width: calc( 12.5% - 8px)`  it fits 7 items on a row. So the math didn't seem to make sense to me and I want to avoid rounding errors. Another example would be on min-width: 768px screen views, with children having `width: calc (calc( 16.66666667% - 7px )`:
![image](https://github.com/Liquipedia/Lua-Modules/assets/105433238/adb74c74-1429-4b57-af89-8640dd97260a)
As you can see the calculations are off and using `8px` would fits 6 items on a row.
I also went and tried `display: grid`, but since the last row of cards need to be centered it wasn't easily done with a grid.

Reply:
I can up with this solution because this is how bootstrap does it with their columns

Reply:
Alright I see, then would just do this:
```suggestion
				padding: calc( @_gutter-x * 0.5 );
```

You'll get the same result and it is cleaner